ionic-template/views/SubSections.js

```diff
@@ -10,7 +10,7 @@ import { IonItem, IonLabel, IonList, IonListHeader } from '@ionic/react'
 // import * as apiActions from 'api-actions'
 
 const propTypes = {
-  // %subSections%: PropTypes.arrayOf(Types.%subSection%),
+  // %subSections%: PropTypes.arrayOf(Types.%SubSection%),
```

Comment:
Types should follow the JS convention of beginning with a capital letter (class format).

Reply:
Hmm, should they? Can you send me a reference? I've never followed this convention.

Reply:
These are a bit strange. They are constant definitions but they are NOT constants. They are objects that represent a type, used for type checking. See: https://google.github.io/styleguide/jsguide.html#naming-rules-by-identifier-type. In my mind, these are similar to classes. The current convention of naming these starting with lower-case didn't really highlight their "typeness" to me. Rather, this convention made them look like locally defined constant variables, which they are not.

Reply:
We should paint the shed blue! That link is helpful. Let's do it.